packages/cli-lib/lib/read.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+const path = require('path');
+
+const STAT_TYPES = {
+  FILE: 'file',
+  SYMBOLIC_LINK: 'symlink',
+  DIRECTORY: 'dir',
+};
+
+const generateFilePromise = (dir, file) => {
+  return new Promise((resolve, reject) => {
+    const filepath = path.join(dir, file);
+    fs.lstat(filepath, (error, stats) => {
+      if (error) {
+        reject(error);
+      }
+      let type;
+      if (stats.isSymbolicLink()) {
+        type = STAT_TYPES.SYMBOLIC_LINK;
+      } else if (stats.isDirectory()) {
+        type = STAT_TYPES.DIRECTORY;
+      } else if (stats.isFile()) {
+        type = STAT_TYPES.FILE;
+      }
+      resolve({ filepath, type });
+    });
+  });
+};
+
+const formatFilesData = filesData => {
```

Comment:
Maybe rename and include `flatten` or `flattenWithoutSymlinks`?

Reply:
Done in b12426c